packages/orbit-components/package.json

```diff
@@ -83,7 +83,85 @@
     "react-uid": "^2.3.1"
   },
   "devDependencies": {
+    "@adeira/babel-preset-adeira": "^3.0.0",
+    "@babel/cli": "^7.12.8",
+    "@babel/core": "^7.12.10",
+    "@babel/generator": "^7.13.16",
+    "@babel/node": "^7.14.2",
+    "@babel/plugin-proposal-class-properties": "^7.14.5",
+    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.14.5",
+    "@babel/plugin-proposal-object-rest-spread": "^7.15.6",
+    "@babel/plugin-proposal-private-methods": "^7.14.5",
+    "@babel/plugin-transform-runtime": "^7.15.0",
+    "@babel/preset-env": "^7.14.0",
+    "@babel/preset-flow": "^7.12.13",
+    "@babel/preset-react": "^7.12.13",
+    "@babel/preset-typescript": "^7.13.0",
+    "@babel/runtime": "^7.11.2",
+    "@babel/types": "^7.13.0",
+    "@emotion/core": "^11.0.0",
+    "@kiwicom/browserslist-config": "^1.0.0",
+    "@kiwicom/orbit-components": "file:.",
```

Comment:
I'm a bit nervous about self-referencing `@kiwicom/orbit-components` like this, the goal was to make ESLint accept `@kiwicom/orbit-components` imports in `__examples__` and `__testfixtures__` work. I checked and this doesn't cause an infinite nesting nesting within `packages/orbit-components/node_modules/@kiwicom/orbit-components`, it only copies necessary files.


This is `devDependencies` so I doubt something can go wrong, and it's only temporary anyway, we'll eventually move examples to `docs`, and we can move codemods to a separate workspace.